tests/py3/utils.py

```diff
@@ -1,6 +1,9 @@
 import json
 from httpx import Response
-from httpx.dispatch.base import AsyncDispatcher
+try:
+    from httpx._dispatch.base import AsyncDispatcher
+except ImportError:
+    from httpx.dispatch.base import AsyncDispatcher
```

Comment:
This module will be changed again in v0.13. That's why I'd like to keep our current implementation.

Reply:
An alternative to using the private dispatch API here would be to implement the stubs as either ASGI or WSGI applications, and use `Client(app=wsgi_app)` or `AsyncClient(app=asgi_app)`.